### ISSUE TITLE
Fieldalignment/govet does not work

### ISSUE THREAD

**Issue by lukasbash** [golangci/golangci-lint#2466]
### Welcome

- [X] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [X] Yes, I've searched similar issues on GitHub and didn't find any.
- [X] Yes, I've included all information below (version, config, etc).
- [X] Yes, I've tried with the standalone linter if available. (https://golangci-lint.run/usage/linters/)

### Description of the problem

I enabled the `govet` linter in my configuration. The command line tool does not report any optimisations. (I doubt that there are none).

### Version of golangci-lint

<details>

```console
$ golangci-lint --version
golangci-lint has version v1.43.0
```

</details>

### Configuration file

<details>

```console
$ cat .golangci.yml
run:
  # Allow multiple parallel golangci-lint instances running.
  # If false (default) - golangci-lint acquires file lock on start.
  allow-parallel-runners: true

output:
  # sorts results by: filepath, line and column
  sort-results: true

linters:
  enable:
    - godox
    - govet
    - revive
    - whitespace
    - wsl
    - stylecheck

linter-settings:
  govet:
    # report about shadowed variables
    check-shadowing: true

    # enable or disable analyzers by name
    # run `go tool vet help` to see all analyzers
    enable-all: true
```

</details>

### Go environment

<details>

```console
$ go version && go env
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/Users/xx/Library/Caches/go-build"
GOENV="/Users/xx/Library/Application Support/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOINSECURE=""
GOMODCACHE="/Users/xx/Packages/go/pkg/mod"
GONOPROXY=""
GONOSUMDB=""
GOOS="darwin"
GOPATH="/Users/xx/Packages/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/local/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/darwin_amd64"
GOVCS=""
GOVERSION="go1.16"
GCCGO="gccgo"
AR="ar"
CC="clang"
CXX="clang++"
CGO_ENABLED="1"
GOMOD="/Users/xx/Projects/test/go.mod"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -arch x86_64 -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/_n/lksgzj492pb2mqyvxsyq1s280000gn/T/go-build1590413801=/tmp/go-build -gno-record-gcc-switches -fno-common"
```

</details>

### Verbose output of running

<details>

```console
$ golangci-lint cache clean
$ golangci-lint run -v
# paste output here
# irrelevant, too many errors, none of them include govet
```

</details>

### Code example or link to a public repository

<details>

```go
// add your code here
```

</details>

**Comment by ldez** [comment]
Hello,

`fieldalignment` analyzer works well, but you have a typo in your configuration: it's "linter**s**-settings" and not "linter-settings".

```
linters-settings:
  govet:
    enable:
      - fieldalignment

linters:
  disable-all: true
  enable:
    - govet
```

https://golangci-lint.run/usage/linters/#govet
